internal/cmd/gocdk/launch.go

```diff
@@ -124,6 +130,22 @@ func newLauncher(ctx context.Context, pctx *processContext, launcherName string)
 			dockerEnv: pctx.env,
 			dockerDir: pctx.workdir,
 		}, nil
+	case "cloudrun":
+		creds, err := pctx.gcpCredentials(ctx)
+		if err != nil {
+			return nil, xerrors.Errorf("prepare cloudrun launcher: %w", err)
+		}
+		httpClient, _ := gcp.NewHTTPClient(http.DefaultTransport, creds.TokenSource)
+		runService, err := cloudrun.NewService(ctx, option.WithHTTPClient(&httpClient.Client))
+		if err != nil {
+			return nil, xerrors.Errorf("prepare cloudrun launcher: %w", err)
+		}
+		return &cloudRunLauncher{
+			logger:    logger,
```

Comment:
Kind of same question as before: if `logger`, `dockerEnv`, and `dockerDir` are going to be needed by every launcher, why not pass them in `LaunchInput`?

Reply:
SGTM.